### DISCUSSION TITLE
Simple user plugin

### DISCUSSION THREAD

**Discussion by earonesty** [FlowiseAI/Flowise#1607] [Ideas]
Sign-in with oauth, (github, etc), just trust the userid you get back, no need to maintain a database of users.   Then you can link that to a chain that supports conversation history, for example.  So users can see their own conversations, and not everyone's.   All the conversation history stuff should have a link to various kinds of auth tools - then anyone can make an auth plugin that does whatever it needs to do to get the downstream tool its userid.

**Comment by hieronymusde** [comment]
Relevant to all external services. I have a vectorstore behind oauth.
